README.md

```diff
@@ -30,6 +30,8 @@ conda activate pudl-usage-metrics
 
 The ETL uses [ipinfo](https://ipinfo.io/) to geocode ip addresses. You need to obtain an ipinfo API token and store it in the `IPINFO_TOKEN` environment variable.
 
+If you want to take advantage of caching raw logs, rather than redownloading them for each run, you can set the optional ``DATA_DIR`` environment variable. If this is not set, the script will save files to a temporary directory by default. This is highly recommended to avoid unnecessary egress charges.
```

Comment:
Out of curiosity, how much data is actually egressed?

Also, @bendnorman , are the log buckets covered under the Open Data situation or do they cost normal egress fees?

Reply:
We're downloading from GCS, not AWS, so I don't think the Open Data situation would be relevant here. Even if it isn't super costly, worth avoiding as much as we can I think.

Reply:
Oh, right. I keep forgetting we mirror the bucket into GCS. I guess we could download the logs directly from the AWS bucket so long as we do it before the logs get deleted :) But that's all not very important.